### PR TITLE
CRUD Menu

### PR DESCRIPTION
**Pull request by Chuvan98**
My first try creating a CRUD

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+DB_CONN_STRING="mongodb://127.0.0.1:27017/"
+# DB_CONN_STRING="mongodb://localhost:27017/"
```
Comment (MelMuro) [inline .env]: Please don't leave commented code

File: src/db/client.ts [hunk 2]
```diff
@@ -10,11 +10,12 @@ export const connectToDb = async () => {
     try {
         const mongoClient = new MongoClient(process.env.DB_CONN_STRING || "");
         await mongoClient.connect();
-
+        console.log('Conexión exitosa a la base de datos');
```
Comment (MelMuro) [inline src/db/client.ts]: Remove console.log statements

**Review by MelMuro — APPROVED** [review]
Looks good to me, I only have a couple of nitpick comments.

File: src/db/client.ts [hunk 3]
```diff
@@ -10,11 +10,12 @@ export const connectToDb = async () => {
...
         const db = mongoClient.db(process.env.DB_NAME);
         dbCollections.Menus = db.collection(process.env.DB_MENUS_COLLECTION || "");
         dbCollections.Restaurants = db.collection(process.env.DB_RESTAURANTS_COLLECTION || "")
     } catch (error) {
+        console.log('error');
```
Comment (Nolan-mjns) [inline src/db/client.ts]: console log with text and the error may be better in the same line being only one console log if you need it

File: src/routes/menus/menu.router.ts [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        res.status(500).send('Error Dishes')
+    }
+});
+
+//Get Dishes for Anywhere
+menusRouter.get('/:dish', async (req: Request, res: Response) => {
+    try {
+        const name = req.params.dish.toLowerCase();
+        const menu = await dbCollections.Menus?.findOne<Menu>({
+            'dishes.name': { $regex: new RegExp(`${name}`, 'i') }
```
Comment (Nolan-mjns) [inline src/routes/menus/menu.router.ts:27]: is the Regex for the blank spaces in dish name?

**Review by Nolan-mjns — COMMENTED** [review]
Doubts

File: src/routes/menus/menu.router.ts (hunk 4 above)
Comment (Chuvan98) [reply src/routes/menus/menu.router.ts:27]: I don't Know :O

**Pull request merged**